src/__tests__/pages/sites-and-applications.test.tsx

```diff
@@ -389,60 +406,25 @@ describe('Sites and Applications page', () => {
           expect(collectionsAdded).toBe(true)
         })
 
-        it('cannot select more than the max number of collections', async () => {
+        test('cannot select more than the max number of collections', async () => {
           const user = userEvent.setup({
             advanceTimers: jest.advanceTimersByTime,
           })
 
-          await user.click(
-            screen.getByRole('button', {
-              name: 'Select multiple collections',
-            })
-          )
-
-          expect(
-            screen.getByRole('button', { name: 'Add selected' })
-          ).toBeDisabled()
-          expect(screen.getByText('0 collections selected')).toBeInTheDocument()
-          expect(
-            screen.getByText('(3 of 25 possible remaining)')
-          ).toBeInTheDocument()
-
-          await user.click(
-            screen.getByRole('button', {
-              name: 'Select collection Example Collection 1',
-            })
-          )
-          expect(screen.getByText('1 collection selected')).toBeInTheDocument()
-          expect(
-            screen.getByText('(2 of 25 possible remaining)')
-          ).toBeInTheDocument()
-
-          await user.click(
-            screen.getByRole('button', {
-              name: 'Select collection Example Collection 2',
-            })
+          renderWithAuthAndApollo(
+            <SitesAndApplications
+              collections={mockCMSCollections}
+              bookmarks={mockCMSBookmarks}
+            />,
+            { portalUser: portalUserCollectionLimit },
+            sitesAndAppsMock
           )
-          expect(screen.getByText('2 collections selected')).toBeInTheDocument()
-          expect(
-            screen.getByText('(1 of 25 possible remaining)')
-          ).toBeInTheDocument()
-
-          expect(
-            screen.queryByRole('button', {
-              name: 'Select collection Example Collection 4',
-            })
-          ).toBeInTheDocument()
 
           await user.click(
             screen.getByRole('button', {
-              name: 'Select collection Example Collection 3',
+              name: 'Select multiple collections',
             })
           )
-          expect(screen.getByText('3 collections selected')).toBeInTheDocument()
-          expect(
-            screen.getByText('(0 of 25 possible remaining)')
-          ).toBeInTheDocument()
```

Comment:
Question: Did you intend to remove all the checks around the max # of collections or was that inadvertent?

Reply:
I think we should add this back, I'm going to do that today.

Reply:
I looked and this is covered just differently than before. I added a couple checks but I understand the test better now.